sdk-utils/src/main/kotlin/com/rakuten/tech/mobile/sdkutils/StringExtension.kt

```diff
@@ -106,4 +108,22 @@ object StringExtension {
             -1
         }
     }
+
+    /**
+     * Get the URL encoded value of the string for charset "UTF-8".
+     *
+     * @return UTF-8 URL encoded value
+     */
+    fun String.getUTF8UrlEncoded(): String {
+        return URLEncoder.encode(this, StandardCharsets.UTF_8.displayName())
```

Comment:
It seems that `URLEncoder.encode` can throw `UnsupportedEncodingException`, maybe better to wrap it in try-catch block?

Reply:
updated